protocol-units/bridge/contracts/src/AtomicBridgeInitator.sol

```diff
@@ -83,8 +97,19 @@ contract AtomicBridgeInitiator is IAtomicBridgeInitiator, Initializable {
         if (bridgeTransfer.state != MessageState.INITIALIZED) revert BridgeTransferStateNotInitialized();
         if (block.number < bridgeTransfer.timeLock) revert TimeLockNotExpired();
         bridgeTransfer.state = MessageState.REFUNDED;
+        // Decrease balance of originator and transfer WETH back to originator
+        balances[bridgeTransfer.originator] -= bridgeTransfer.amount;
         if (!weth.transfer(bridgeTransfer.originator, bridgeTransfer.amount)) revert WETHTransferFailed();
 
         emit BridgeTransferRefunded(bridgeTransferId);
     }
+
+    // Counterparty contract to withdraw WETH for originator
+    function withdrawWETH(address originator, uint256 amount) external {
```

Comment:
Maybe call this the receipient, as this is how its called from the Counterpart contract.

Reply:
yep, better.